cluster-state-service/handler/reconcile/loader/taskloader.go

```diff
@@ -148,6 +153,31 @@ func (loader taskLoader) getTasksFromECS(cluster *string) ([]types.Task, error)
 	return tasks, nil
 }
 
+// getTaskARNsFromECS gets a list of the task ARNs from ECS for both running and stopped tasks.
+// ECS currently stores stopped tasks for about 1 hour.
```

Comment:
How do we know this is stored for 1 hour? If we know this by experimentation and ECS does not garantee this anywhere in it's documentation, this note here can get outdated easily.